Source/Plugin.BLE/Windows/Device.cs

```diff
@@ -238,7 +239,23 @@ public override void Dispose()
 
         protected override DeviceBondState GetBondState()
         {
-            return DeviceBondState.NotSupported;
+            try
+            {
+                DeviceInformation deviceInformation = DeviceInformation.CreateFromIdAsync(NativeDevice.DeviceId).AsTask().Result;
+                if (deviceInformation.Pairing.IsPaired)
+                {
+                    return DeviceBondState.Bonded;
+                }
+                else
+                {
+                    return DeviceBondState.NotBonded;
+                }
```

Comment:
Looks good, but I guess this could written a bit more compact:

`return deviceInformation.Pairing.IsPaired ? DeviceBondState.Bonded : DeviceBondState.NotBonded`

One line instead of eight ;)

Reply:
Yes - we can do that.
Done.

Reply:
thx